invenio_app_ils/circulation/utils.py

```diff
@@ -67,7 +67,7 @@ def circulation_is_item_available(item_pid):
     """Check if item is available."""
     item = Item.get_record_by_pid(item_pid)
     if item:
-        return item["status"] == "LOANABLE"
+        return item["status"] in ["LOANABLE", "MISSING"]
```

Comment:
I think we should not change this. Is this really needed? Maybe better discuss it.

Reply:
Changed it back as discussed, item status is being modified before this check.